packages/acs-ui-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internal/acs-ui-common",
-  "version": "1.0.0-beta.2",
+  "version": "1.0.0-beta.3",
```

Comment:
Interesting! Last time there was no change in this package so it wasn't bumped, we do not need to do anything about this for this release but next release (beta.5) I'll update to update all packages in lockstep (i.e. ensure all packlets have the same version)

Reply:
note: there's no issue to them being different, but given the final package is re-exporting functionality from the packlets it makes sense semantically for them to all be at the same version as the npm package

Reply:
thanks for explaining, @JamesBurnside  :)

Reply:
There will be an issue once we start reading the version number from _somewhere_ to report to telemetry.